docs/configuration.md

```diff
@@ -1,6 +1,6 @@
-Keeper's default behavior with no configuration will enable it for _all_
-androidTest variants. This may not be what you want for your actual production builds that you plan
-to distribute.
+Keeper's default behavior with no configuration will enable it for only buildTypes which use
+minifyEnabled = true. This may not be what you want for your actual production builds that you
```

Comment:
```suggestion
Keeper's default behavior with no configuration will enable it for only buildTypes that set
`minifyEnabled` to true. This may not be what you want for your actual production builds that you
```